examples/jsm/nodes/core/StackNode.js

```diff
@@ -39,7 +39,7 @@ class StackNode extends Node {
 
 	if( boolNode, method ) {
 
-		const methodNode = shader( method );
+		const methodNode = new ShaderNode( method );
```

Comment:
Why do we change this?

Reply:
`shader` was deprecated, I think that soon `ShaderNode` will be replaced by `TSL` function `tslFn`, but for this we need to create `ShaderNodeCall`, otherwise the function is called without `builder` and `stack` parameters for some cases.